xilem_masonry/src/view/button.rs

```diff
@@ -34,7 +34,9 @@ use crate::{Pod, ViewCtx, WidgetView};
 /// To create a simple button with styled text:
 ///
 /// ```
-/// use xilem::{view::{button, label}, FontWeight};
+/// # use xilem_masonry as xilem;
```

Comment:
```suggestion
```

Can we avoid these changes in all the doc-examples? They shouldn't be necessary, and will likely be out of sync at some time. Btw. (as I think I was the author of at least a few of those examples), I intentionally have put the `use` statements all on one line (if not unnecessarily long) to shift focus to the actual interesting part, as these are often autocompleted/imported anyway.